assets/dev/js/editor/document/backwards-compatibility.js

```diff
@@ -1,49 +1,10 @@
 export default class BackwardsCompatibility {
```

Comment:
Please check with @iNewLegend if we can completely remove this file

Reply:
Checked with him, file was removed